include/rocksdb/options.h

```diff
@@ -412,6 +412,8 @@ struct DBOptions {
   // This parameter should be set to true while storing data to
   // filesystem like ext3 that can lose files after a reboot.
   // Default: false
+  // Note: on many platforms fdatasync is defined as fsync, so this parameter 
```

Comment:
There is currently no use_data_sync in the code base. Can you tell more how would it affect the added note?